### ISSUE TITLE
More graceful failure when languages not found

### ISSUE THREAD

**Issue by duncanspumpkin** [OpenRCT2/OpenLoco#258]
Currently when you do not have the language files it quits without any feedback to the user as to what is wrong.

**Comment by AaronVanGeffen** [comment]
Yeah, it only outputs an error to `stdout`. We should probably show a message box or something.
